CONTRIBUTING.md

```diff
@@ -79,13 +79,35 @@ For example:
 
 Over time, we should track which items are useful and which don't get any traffic. Also, as Apify Docs expand, we may need to add more cards and update which articles we link to.
 
-## Local setup
+## Installation and setup
```

Comment:
@B4nan I'd appreciate if you could take a look whether I got these sections right.

Reply:
sure thing